x/ethbridge/keeper/pauser.go

```diff
@@ -0,0 +1,23 @@
+package keeper
+
+import (
+	"github.com/Sifchain/sifnode/x/ethbridge/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (k Keeper) SetPauser(ctx sdk.Context, pauser *types.Pauser) {
```

Comment:
This needs to be renamed, it is not storing who is a pauser but the status of the chain as being paused or not.